app/src/main/res/values-night/themes.xml

```diff
@@ -14,22 +14,41 @@
 
         <!-- Customize your theme here. -->
 
-
     </style>
-
-
-
-    <style name="textSubTitle">
+    <style name="TextSubTitle">
         <item name="android:fontFamily">@font/roboto_light</item>
         <item name="android:textColor">#9A9A9A</item>
         <item name="android:textSize">@dimen/textMedium</item>
     </style>
-
-    <style name="circle">
+    <style name="TextKitchens">
+        <item name="android:layout_marginStart">@dimen/spaceMedium</item>
+        <item name="android:layout_marginBottom">@dimen/spaceSmall</item>
+        <item name="fontFamily">@font/roboto_bold</item>
+        <item name="android:textSize">@dimen/textXLarge</item>
+        <item name="android:textColor">@color/white</item>
+    </style>
+    <style name="TextFood">
+        <item name="android:layout_marginHorizontal">@dimen/spaceSmall</item>
```

Comment:
use android:layout_marginStart and marginEnd instead of android:layout_marginHorizontal